src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisCacheWrapper.cs

```diff
@@ -11,21 +10,21 @@ namespace OrchardCore.Redis.Services;
 /// <remarks>
 /// This is done by not disposing the `RedisCache` instance which would otherwise close the redis connection.
 /// </remarks>
-public sealed class RedisCacheWrapper : IDistributedCache
+internal sealed class RedisCacheWrapper : IDistributedCache
 {
-    private readonly RedisCache _cache;
+    private readonly IDistributedCache _cache;
 
-    public RedisCacheWrapper(IOptions<RedisCacheOptions> optionsAccessor) => _cache = new RedisCache(optionsAccessor);
+    public RedisCacheWrapper(IDistributedCache cache) => _cache = cache ?? throw new ArgumentNullException(nameof(cache));
```

Comment:
I'm not sure why you changed the parameter type?

Reply:
Because `RedisCache` cannot be easily mocked for unit testing.